### DISCUSSION TITLE
question: clientIp on docker

### DISCUSSION THREAD

**Discussion by ramarro123** [0xERR0R/blocky#1248] [General]
hi, i am running blocky on a docker image forwarding a few port on host, including 53, 443, 843

docker host ip is 192.168.1.10

aside some minor issue, i am able to use it.
a dig report (using 192.168.1.10)
("..BLOCKED (ads)")
 using
adservice.google.com

log report
2023-11-14 19:14:37	172.17.0.1	none	4		adservice.google.com.				A	spx01-blocky1

the ip seems the clientIp of the docker_host_if not my host that should be something like 192.168.1.83

my knowledge in that matter is almost zero, so feel free to close the ticket if it's not the appropriate way to ask for support.

thanks for helping!

**Comment by ThinkChaos** [comment]
You need to disable source NAT. There are multiple solutions for that. Searching source NAT/SNAT should help.
